### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wardbekker**
updated required Go version for building loki

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Refer to [CONTRIBUTING.md](CONTRIBUTING.md)
 
 Loki can be run in a single host, no-dependencies mode using the following commands.
 
-You need `go` [v1.10+](https://golang.org/dl/) installed locally.
+You need `go` [v1.14+](https://golang.org/dl/) installed locally.
```
Comment (slim-bean) [inline README.md:102]: I'm not sure that we strictly require 1.14? For that matter I don't know if we strictly require 1.10 or later.

I wonder if we should reword this to something like:

```
You need `go`, we recommend using the version found in [our build Dockerfile](https://github.com/grafana/loki/blob/master/loki-build-image/Dockerfile)
```

Comment (wardbekker) [reply README.md:102]: I was not able to build Loki with Go version 1.13.x. Referring to the docker file version would keep these instructions evergreen.

**Review by slim-bean — APPROVED** [review]
Looks good! Thanks Ward!

File: README.md (hunk 1 above)
Comment (slim-bean) [reply README.md]: Good to know, not sure why this would be!?
